x-pack/plugins/infra/public/pages/metrics/hosts/hosts_content.tsx

```diff
@@ -6,81 +6,101 @@
  */
 
 import type { Query, TimeRange } from '@kbn/es-query';
-import { useKibana } from '@kbn/kibana-react-plugin/public';
 import { i18n } from '@kbn/i18n';
 import React, { useState, useCallback } from 'react';
 import { SearchBar } from '@kbn/unified-search-plugin/public';
+import { EuiSpacer } from '@elastic/eui';
+import { NoData } from '../../../components/empty_states';
 import { InfraLoadingPanel } from '../../../components/loading';
 import { useMetricsDataViewContext } from './hooks/use_data_view';
 import { HostsTable } from './components/hosts_table';
-import { InfraClientStartDeps } from '../../../types';
 import { useSourceContext } from '../../../containers/metrics_source';
+import { useSnapshot } from '../inventory_view/hooks/use_snaphot';
+import { SnapshotMetricType } from '../../../../common/inventory_models/types';
```

Comment:
nit (just type import):
```suggestion
import type { SnapshotMetricType } from '../../../../common/inventory_models/types';
```